.github/workflows/update.yml

```
@@ -4,18 +4,48 @@ on:
-          source: deno.json
+          deno-version: ${{ matrix.deno.version }}
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: npm
+
+      - name: Audit node dependencies
+        run: npm audit fix --force
+
+      - name: Check 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)